manager/assets/ext3/ext-all-debug.js

```diff
@@ -40437,7 +40437,7 @@ Ext.form.Field = Ext.extend(Ext.BoxComponent,  {
         if(v === this.emptyText || v === undefined){
             v = '';
         }
-        return v;
+        return Ext.util.Format.htmlEncode(v);
```

Comment:
While this is clever, I don't think we should apply htmlEncode to the getValue method in this way (assuming that's where you added that, GitHub wont show me more context because it's a too large file).

There's nothing wrong with a value including some good ol' HTML (we don't want all content to suddenly be html encoded for example, people do use RTEs), as long as it's is escaped based on the specific places that uses it. Inserting a value into a header is such a place that needs special treatment.

Reply:
Oh, i missed that part

Reply:
Looks like this still needs to be reverted.

Reply:
@Mark-H I reverted and fixed